spec.html

```diff
@@ -41704,7 +41528,7 @@ <h1>The %AsyncFromSyncIteratorPrototype% Object</h1>
           <h1>%AsyncFromSyncIteratorPrototype%.next ( [ _value_ ] )</h1>
           <emu-alg>
             1. Let _O_ be the *this* value.
-            1. Assert: Type(_O_) is Object and _O_ has a [[SyncIteratorRecord]] internal slot.
+            1. Assert: Type(_O_) is Object with a [[SyncIteratorRecord]] internal slot.
```

Comment:
This change makes no grammatical sense. Ditto the other occurrences.

Reply:
Oops, I thought I fixed these before opening the PR.